Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -2296,21 +2312,25 @@ def fetch_last_emails(
         assert first_fetch_datetime is not None
         first_fetch_ews_datetime = EWSDateTime.from_datetime(first_fetch_datetime.replace(tzinfo=tz))
         qs = qs.filter(last_modified_time__gte=first_fetch_ews_datetime)
-    qs = qs.filter().only(*[x.name for x in Message.FIELDS])
+    qs = qs.filter().only(*[x.name for x in Message.FIELDS if x.name != 'mime_content'])
     qs = qs.filter().order_by("datetime_received")
-
     result = []
     exclude_ids = exclude_ids if exclude_ids else set()
     demisto.debug(f'{APP_NAME} - Exclude ID list: {exclude_ids}')
+    qs.chunk_size = min(client.max_fetch, 100)
+    qs.page_size = min(client.max_fetch, 100)
```

Comment:
Does the user care about these defaults? (100)

Reply:
100 is the given default at the moment from microsoft. I guess we can add parameters with warning signs for the customers. Should this be made into params?

Reply:
How can it affect customers? Maybe just a disclaimer in docs

Reply:
this shouldnt affect customers, only performance. Page size and chunk size are instructions to the library for how to split up the http requests. Its an implementation detail
Maybe Im not understanding the question?

Reply:
👍